src/config.template.docker.cjs

```diff
@@ -80,15 +80,25 @@ module.exports = {
 	outputDir: "/cross-seeds",
 
 	/**
-	 * Whether to search for single episode torrents
+	 * 
+	 * To search for all episodes, use (includeEpisodes: true, includeNonVideos: true)
```

Comment:
I think this blurb still makes more sense in its old form above includeNonVideos.

Reply:
moved